apps/fabric-website/src/components/App/App.tsx

```diff
@@ -58,7 +58,10 @@ export class App extends React.Component<IAppProps, any> {
 
   public componentWillReceiveProps(nextProps: IAppProps): void {
     if (nextProps && nextProps.children !== this.props.children) {
-      document.body.scrollTop = document.documentElement.scrollTop = 0;
+      const hash = this._extractAnchorLink(window.location.hash);
+      if (hash.substring(0, 11) === '/components') {
```

Comment:
Why substring vs [String.prototype.indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf?

Reply:
No reason! I think `indexOf` is actually more intuitive, just changed it